pwnlib/tubes/ssh.py

```diff
@@ -1177,7 +1177,7 @@ def getenv(self, variable, **kwargs):
 ''' % variable
 
         with context.local(log_level='error'):
-            python = self.which('python')
+            python = self.which('python') or self.which('python2.7') or self.which('python3')
```

Comment:
python3 should be preferred to python2
```suggestion
            python = self.which('python') or self.which('python3') or self.which('python2')
```